packages/api/amplify_api/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -221,50 +253,131 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
   }
 
   // handle connection errors on the web socket channel
-  Stream<WebSocketState> _connectionError(
-    ConnectionErrorEvent event,
-  ) async* {
-    yield _currentState.failed();
-
+  Stream<WebSocketState> _connectionError(ConnectionErrorEvent event) async* {
     const exception = ApiException(
       'Error occurred while connecting to the websocket',
     );
 
-    await _sendExceptionToBlocs(exception);
+    yield _currentState.failed(exception);
```

Comment:
Since the first line of _shutdownWithException emits a failed state we probably don't need this here.